### ISSUE TITLE
[MS Edge] When cooperate with other extension MS edge fail to load some pages

### ISSUE THREAD

**Issue by lychichem** [Tampermonkey/tampermonkey#319]
Sorry for the empty bexmcause I used my mobile phone to open this issue.

**Comment by lychichem** [comment]
MS edge's about: Page and ms-appx-web:// pages cannot load. For example, cooperate with adblock.

**Comment by lychichem** [comment]
Some of my fellows can reproduce it and they all use tampermonkey, so I think t this is a bug of tampermonkey, and the solution is simple, just make tampermonkey disabled on ms-appx-web:// pages is ok.

**Comment by derjanb** [comment]
I have no idea how to fix or even workaround this in Tampermonkey. Most probably this needs to be fixed at MS Edge.
https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/8774109/

**Comment by lychichem** [comment]
What if we make Tampermonkey don't work on Edge's internal pages, like what Adblock does?

**Comment by lychichem** [comment]
What if we make Tampermonkey don't work on Edge's internal pages, like what Adblock does?

**Comment by derjanb** [comment]
Tampermonkey already doesn't do anything at that pages.

**Comment by lychichem** [comment]
Confirm the issue is fixed at Tampermonkey for Edge 4.2.5303

**Comment by lychichem** [comment]
Sorry for that but I need to reopen this issue because tampermonkey still breaks Adblock's "show resource list" page.

**Comment by derjanb** [comment]
This is working fine here. I guess a browser re-start doesn't solve the issue?

**Comment by lychichem** [comment]
Well, the issue is partly fixed. Now pages belong to Edge itself is not broken, but other extensions pages (here I use Adblock's "show resource list" page as an example) may broken. In my situation, the page says "no resource was send" if TM is enabled and works fine when TM is disabled.

**Comment by derjanb** [comment]
Ok, I've found "show resource list" page and reproduced the issue. (The page is only visible in advanced settings mode.)

Hmm, it looks like extensions can interfere each other. For example I've noticed that with AdBlock installed it's sometimes not possible to install a script, but they are downloaded...

However, after installing and uninstalling both extensions it sometimes works as expected. I'll report this to the MS Edge developers.
